app/main.go

```diff
@@ -217,23 +173,25 @@ func (app *SpacemeshApp) cleanup(ctx *cli.Context) error {
 	return nil
 }
 
-func (app *SpacemeshApp) startSpacemeshNode(ctx *cli.Context) error {
+func (app *SpacemeshApp) startSpacemeshNode(cmd *cobra.Command, args []string) {
 	log.Info("Starting local node...")
-	port := *nodeparams.LocalTCPPortFlag.Destination
+	fmt.Printf("Currently seing tcp port %v \n", app)
+	port := app.Config.P2P.TCPPort
 	address := fmt.Sprintf("0.0.0.0:%d", port)
 
 	// start a new node passing the app-wide node config values and persist it to store
 	// so future sessions use the same local node id
-	node, err := p2p.NewLocalNode(address, nodeparams.ConfigValues, true)
+	node, err := p2p.NewLocalNode(address, app.Config.P2P, true)
 	if err != nil {
-		return err
+		fmt.Printf("%v", err)
```

Comment:
- again, please use `log`. 
- remove comment
also this means node failed to start. we should print the error and stop the app.